static/scss/certificates/certificate.scss

```diff
@@ -289,3 +275,144 @@ $text-gray-light: #d2d0d1;
     width: 25px;
   }
 }
+
+@media print {
+  a {
+    color: black !important;
+  }
+
+  .notifications.site-wide {
+    display: none !important;
+  }
+
+  .no-print {
+    display: none;
+  }
+
+  .full-width-print {
+    max-width: 100%;
+  }
+
+  .certify-by-row {
+    display: block;
+
+    &:after {
+      content: "";
+      display: block;
+      clear: both;
+    }
+
+    .certify-by {
+      float: left;
+      width: 33.3333%;
+
+      &.signature-1 {
+        float: none;
+        margin: 0 auto;
+      }
+
+      &.signature-2 {
+        width: 30%;
+
+        &:first-child {
+          margin-left: 20%;
+        }
+      }
+
+      &.signature-4 {
+        width: 25%;
+      }
+
+      &.signature-5 {
+        width: 20%;
+      }
+    }
+  }
+
+  @page {
+    size: auto;
```

Comment:
I think this is where we can add the landscape setting

Reply:
actually, I choose
```
@page {
    size: auto;
  }

```
instead of
```
@page {
    size: landscape;
  }

```
to prevent the cut-offs in chrome-portrait mode.